docs/src/main/asciidoc/_galleon/Galleon_layers.adoc

```diff
@@ -0,0 +1,863 @@
+[[wildfly_defined_galleon_layers]]
+=== WildFly Layers
+
+In the tables below we provide basic information about all of the layers WildFly provides.
+
+If the capabilities provided by a layer A require capabilities provided by another layer B, then layer A will _depend on_ layer B.
+If you ask for layer A, then Galleon will automatically provide B as well. In some cases A's dependency on B can be _optional_; that
+is A typically comes with B, but can function without it. In this case if you ask for A by default Galleon will provide B as well,
+but you can tell Galleon to _exclude_ B. The tables below detail the various dependency relationships.
+
+Some layers are logical _alternatives_ to other layers. If two layers are alternatives to each other they both provide the same general
+capabilities, but with different implementation characteristics. For example a number of layers provide the capability to cache different
+types of objects. These layers typically come in pairs of alternatives, where one alternative provides local caching, while the other provides
+distributed caching.  If a layer you want has an optional dependency on a layer that has an alternative, you can exclude that dependency
+and instead specify the alternative. If a layer has an alternative the Description column in the tables below will identify it.
+
+[NOTE]
+====
+If the elytron layer is present, security will be handled by the elytron subsystem.
+The undertow subsystem is configured with an _other_ _application-security-domain_ that references the Elytron _ApplicationDomain_ security domain.
+====
+
+[[wildfly-galleon-layers-servers]]
+==== Server Galleon layers
```

Comment:
The title "Server Galleon layers" doesn't feel to me to give the emphasis that this is the starting point.

Maybe something more like "Primary Galleon Layers", "Component Layers" - something to add to the emphasis there is a preferred starting point.

Reply:
I will let @bstansberry answer this one.

Reply:
I struggled with this as well.

Maybe "Foundation" or "Foundational"?

Or "Base Server". And then we need to change the title of the next table from "Basic" to something else.

Reply:
@bstansberry  + 1 on "Foundational Layers".

Reply:
I used Foundational for the first table.